backend/controllers/follows.js

```diff
@@ -48,3 +48,47 @@ module.exports.createFollows = async (req, res) => {
     });
   }
 };
+
+
+module.exports.unfollowUser = async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const unfollowedUserId = req.params.unfollowedUserId;
+
+    if (userId == unfollowedUserId) {
+      return res.status(400).json({
+        message: "Users cannot unfollow themselves",
+      });
+    }
+
+    const count = await Follows.destroy({
+      where: {
+        userId: userId,
+        followingUserId: unfollowedUserId
+      }
+    });
+
+    if (count == 0) {
+      return res.status(400).json({
```

Comment:
No need to return 400 in this case
The purpose of the API has been fulfilled already
We can just ignore such cases